Database/Table Scripts/CreateEventsTableScript.sql

```diff
@@ -0,0 +1,14 @@
+IF (EXISTS (SELECT 1 FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'dbo' AND TABLE_NAME = 'JobEvent'))
+BEGIN
+	DROP TABLE dbo.JobEvent;
+END
+
+CREATE TABLE dbo.JobEvent(
```

Comment:
Let's add one more field:

Canceled - boolean

This will be set to true when someone cancels and it will not be displayed on the calendar